components/OpenSaucedLogo.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react'
+
+interface Props {
+    logo: {
+        title: string,
+        description: string,
+        svgLogo: string,
+        pngLogo: string,
+        isBlackBG: boolean,
+    }
+}
+
+const OpenSaucedLogo:FC<Props> = ({logo}) => {
+    const { title, description, pngLogo, svgLogo, isBlackBG} = logo;
+
+    return (
+    <div className='w-full tablet:max-w-[290px]'>
+        <div className={`flex h-[164px] w-full justify-center items-center ${ isBlackBG ? "bg-black" : "bg-none"} ` }>
+            <div className='h-[100px] w-auto'>
```

Comment:
```suggestion
            <div className='h-24 w-auto'>
```

h-24 = 96px
h-28 = 112px